components/docs-chef-io/content/automate/ha_add_nodes_to_the_deployment.md

```diff
@@ -149,11 +162,14 @@ You can mix and match different services to add nodes across various services.
 
 Once the command executes, it will add the supplied nodes to your automated setup. The changes might take a while.
 
+
 {{< note >}}
 
 - The new node will be configured with the certificates that are already configured in your HA setup.
 - If you had applied unique certificates per node, then the certificates of one of the nodes have been applied by default on the new nodes.
 - If you want to change the certificates for the new nodes, you can manually run the chef-automate cert-rotate [options] command.
+- Removing unreachable nodes while adding nodes is not supported in AWS mode of deployment, you have to use [remove node commands page](/automate/ha_remove_single_node_from_cluster/)
```

Comment:
Replace it with: - Removing unreachable nodes while adding nodes is not supported in AWS mode of deployment; you have to use [remove node commands page](/automate/ha_remove_single_node_from_cluster/)